Update to 6.4

mechagomizilla opened the pull request:
Lots of fixes.

mechagomizilla committed:
Update to 6.2

Lots of fixes.

ortiza5 committed:
Make slight stylistic changes to address bar fixes

ortiza5 committed:
Remove address/search dropdown outline since the field outline is now removed

ortiza5 commented on Mod/sources/address-bar.css:
```diff
@@ -208,8 +221,8 @@ path[d^="M3 4h10v1H3zM3"] {
 .SiteInfoButton.certified {
-  fill: var(--colorAccentFg);
-  color: var(--colorAccentFg);
+  fill: var(--GXbuttonFill);
+  color: var(--GXbuttonFill);
```
This change is more of a personal preference. I believe @beyxnd wanted it to match the URL address, but since the color of all the address bar icons are deviating from Opera GX, there isn't really a right answer.

ortiza5 replied:
@BPZ3, do you have any thoughts on changing the padlock icon to match the other address bar buttons' fill?

ortiza5 commented on Mod/sources/bookmarks-bar.css:
```diff
@@ -19,12 +19,15 @@
 .bookmark-bar button:not(.chevron):before {
   content: unset;
 }
+/* Align titles to look more centered (like Opera and other browsers) */
+.bookmark-bar button .title {
+  margin-bottom: -1px;
+}
```
Does this have any effect on your system? I don't see any difference when I enable/disable this line.

mechagomizilla replied:
I just noticed an issue with scaling. It should be resolved by using -0.5px instead, but I can't test it at the moment.

mechagomizilla replied:
Turns out the issue is not with the titles, but with the images. I will have to find another solution. We can remove this for now.
```suggestion
```

ortiza5 committed:
Make slight stylistic changes to tab bar fixes

mechagomizilla commented:
Yeah, on both my 1080p and 1440p monitors, it does what the comment says. I
think titles in Vivaldi are perfectly centered vertically, while every
other browser lowers them by 1 px since there are fewer characters that use
the bottom section of the lines. It seems to be the most common practice.
Without it, the favicons look a bit low compared to the capital letters
next to them.


> ***@***.**** commented on this pull request.
> ------------------------------
>
> In Mod/sources/bookmarks-bar.css
> <https://github.com/ortiza5/Vivaldi-GX/pull/4#discussion_r1372469312>:
>
> > +/* Align titles to look more centered (like Opera and other browsers) */
> +.bookmark-bar button .title {
> +  margin-bottom: -1px;
> +}
>
> Does this have any effect on your system? I don't see any difference when
> I enable/disable this line.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ortiza5/Vivaldi-GX/pull/4#pullrequestreview-1698581009>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFL43VCULPVRUT6JWW35S2TYBG3UXAVCNFSM6AAAAAA6QBK3SGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOJYGU4DCMBQHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

ortiza5 committed:
Margin works better than padding since there is a background

ortiza5 committed:
Unrelated fix to make the workspace button look better

ortiza5 commented:
@mechagomizilla Are you alright with the small changes I made? It looks good to me, so I am alright with merging.

Thanks @mechagomizilla and @BPZ3 for helping keeping it updated.

ortiza5 commented:
> Yeah, on both my 1080p and 1440p monitors, it does what the comment says. I think titles in Vivaldi are perfectly centered vertically, while every other browser lowers them by 1 px since there are fewer characters that use the bottom section of the lines. It seems to be the most common practice. Without it, the favicons look a bit low compared to the capital letters next to them.

Just to be clear, you are talking about the bookmark bar titles? There is similar CSS for the tabs, which does have an effect on my system, but the bookmark titles don't move for me. Can you try enabling/disabling it with devTools?

![bookmark-titles](https://github.com/ortiza5/Vivaldi-GX/assets/43008712/2527a375-9b01-4699-8528-86fa87e7dee8)

ortiza5 committed:
Fix incorrect background on downloads popup elements

mechagomizilla commented:
> Just to be clear, you are talking about the bookmark bar titles? There is similar CSS for the tabs, which does have an effect on my system, but the bookmark titles don't move for me. Can you try enabling/disabling it with devTools?
> 
> ![bookmark-titles](https://user-images.githubusercontent.com/43008712/278190618-2527a375-9b01-4699-8528-86fa87e7dee8.gif)

Do you perhaps have 150 or 200% display scale? It might explain why you don't see anything. I just recorded this in a different computer than the one I usually use: 
![margin-bottom](https://github.com/ortiza5/Vivaldi-GX/assets/22531540/adbc52a9-ddda-44e9-a1cf-749056e99127)
It's the same for the tab titles.

mechagomizilla commented on Mod/sources/bookmarks-bar.css:
```diff
@@ -19,12 +19,15 @@
 .bookmark-bar button {
   border-radius: var(--radius);
   padding: 0px 5px;
-  cursor: pointer;
```
You can ignore this change if you want, it's just my personal preference.

mechagomizilla commented:
> @mechagomizilla Are you alright with the small changes I made? It looks good to me, so I am alright with merging.
> 
> Thanks @mechagomizilla and @BPZ3 for helping keeping it updated.

This looks alright to me. I already noticed two or 3 more elements that I plan to fix in future pull requests. This was the first pull request I ever made, so I'm still getting the hang of it. I'll make them separate next time.

mechagomizilla commented on Mod/sources/tab-bar.css:
```diff
@@ -253,6 +258,19 @@
+.tab-position .tab .tab-header {
+  flex: 0 0 27px;
+}
+/* Re-align tile to look more centered (like in Opera and other browsers) */
```
Oops, I meant to write "title"
```suggestion
/* Re-align title to look more centered (like in Opera and other browsers) */
```

mechagomizilla commented on Mod/sources/tab-bar.css:
```diff
@@ -253,6 +258,19 @@
+}
+/* Re-align tile to look more centered (like in Opera and other browsers) */
+.tab-position .tab .title {
+  margin-bottom: -1px;
```
```suggestion
  margin-bottom: -0.5px;
```

mechagomizilla committed:
Fixed a typo

mechagomizilla commented on Mod/sources/tab-bar.css:
```diff
@@ -253,6 +258,19 @@
+.tab-position .tab .tab-header {
+  flex: 0 0 27px;
+}
+/* Re-align title to look more centered (like in Opera and other browsers) */
+.tab-position .tab .title {
+  margin-bottom: -1px;
+}
```
```suggestion
```

mechagomizilla committed:
Removed titles realignment

Since this is a problem with the scaling of the images (i.e. favicons), I'm removing this for now.

mechagomizilla commented on Mod/modSettings.css:
```diff
@@ -1,9 +1,10 @@
-  - forum link (new):
-  - last tested with version 6.1.3035.111
+  - forum link (new): https://forum.vivaldi.net/post/686366
+  - last tested with version 6.2.3105.58
```
```suggestion
  - last tested with version 6.4.3160.34
```

mechagomizilla commented on README.md:
```diff
@@ -1,9 +1,10 @@
 An Opera GX style custom UI mod skin for Vivaldi.
 - Originally created by [@gabevilela](https://forum.vivaldi.net/user/gabevilela) with help from [@tam710562](https://forum.vivaldi.net/user/tam710562) on the Vivaldi Forum and modified/updated to its current form by [@nomadic](https://forum.vivaldi.net/user/nomadic)
 - Thanks to [@beyxnd](https://forum.vivaldi.net/user/beyxnd) for testing this mod through development and helping out with finding bugs and making suggestions
+- Update from 6.1 to 6.2 by [@gomizilla](https://forum.vivaldi.net/user/gomizilla) with contributions from [@beyxnd](https://forum.vivaldi.net/user/beyxnd)
```
```suggestion
- Contributions from by [@gomizilla](https://forum.vivaldi.net/user/gomizilla) and [@beyxnd](https://forum.vivaldi.net/user/beyxnd)
```

mechagomizilla committed:
Bump to 6.4

ortiza5 commented:
Think it is good to go for now. Thanks for your work. I will be happy to accept future pull requests if you want to make any. As for making the UI less compact, probably wouldn't merge that in. I actually use CSS on my system to make it more compact 😅

ortiza5 committed:
Removed extra word
